modules/tensor_mechanics/src/materials/ADComputeFiniteStrainElasticStress.C

```diff
@@ -60,3 +60,6 @@ ADComputeFiniteStrainElasticStress<compute_stage>::computeQpStress()
   // Assign value for elastic strain, which is equal to the mechanical strain
   _elastic_strain[_qp] = _mechanical_strain[_qp];
 }
+
+// explicit instantiation is required for AD base classes
+adBaseClass(ADComputeFiniteStrainElasticStress);
```

Comment:
This should not be necessary, as `ADComputeFiniteStrainElasticStress` gets registered and thus instantiated automatically.

Reply:
@dschwen, when I comment out the above line and compile MASTODON, i get the following error:`dyld: Symbol not found: __ZN34ADComputeFiniteStrainElasticStressIL12ComputeStage0EE12initialSetupEv`  in all MASTODON tests. I have tried makeclean in both MOOSE and MASTODON and that doesn't seem to help. The MASTODON material model is in https://github.com/sveerara/mastodon/tree/ad_isoil

Any suggestions on what is causing this error?